docs/services.md

```diff
@@ -14,6 +14,22 @@ The Heroku `Procfile` in the root of the repository determines how to run variou
 The docker image is configured to run these components with `docker run <image> <component>`.
 As with Heroku, the `web` proc is privileged, and is expected to run a web server on port 80, so `docker run <image> web` will run the web service.
 
+## Testing Service Images
+
+To open a shell in a service's docker image, use a commad like the following:
+
+```shell
+docker run -ti --rm <image> bash
```

Comment:
If we are trying to remove the need to look into docker documentation, we can even say `<image_name:image_tag>` instead of `<image>` (but I am being obnoxious as per usual)

Reply:
updated (and fixed "commad")